lib/elixir/test/elixir/option_parser_test.exs

```diff
@@ -256,6 +256,12 @@ defmodule OptionParserTest do
            == {[foo: true, boo: "-"], ["-"], []}
   end
 
+  test "allow nonexistent atom`s" do
+    opts = [switches: [a: :string], allow_nonexistent_atoms: true]
+    assert OptionParser.parse(["--a", "b", "--c", "d", "--option-key-creates-atom", "f"], opts)
```

Comment:
I guess I'll go boy scout style and fix it in whole file. Because I picked this syntax only to match other cases :).